Learning_FEM_1D_step0.ipynb

```diff
@@ -416,7 +420,8 @@
     "$$\\sum_{j=i-1}^{j=i+1} Du_j \\int_{x_{i-1}}^{x_{i+1}} dx \\,\\left( \\frac{db_i}{dx}\\frac{db_j}{dx} \\right) $$\n",
     "\n",
     "Using the piece-wise linear definition of the basis functions, we can compute the integral and obtain\n",
-    "$$ - u_{i-1} \\frac{D}{h_i} + u_{i} \\left(  \\frac{D}{h_i} + \\frac{D}{h_{i+1}}\\right) - u_{i+1} \\frac{D}{h_{i+1}}  $$\n",
+    "$$\\sigma u_{i-1} \\int_{x_{i-1}}^{x_{i}} b_i(x) b_{i-1}(x) \\, dx + \\sigma u_i \\int_{x_{i-1}}^{x_{i+1}} b_i(x) b_i(x) \\, dx + \\sigma u_{i+1} \\int_{x_{i}}^{x_{i+1}} b_i(x) b_{i+1}(x) \\, dx$$\n",
+    "$$\\implies - u_{i-1} \\frac{D}{h_i} + u_{i} \\left(  \\frac{D}{h_i} + \\frac{D}{h_{i+1}}\\right) - u_{i+1} \\frac{D}{h_{i+1}}  $$\n",
```

Comment:
uh ???

Reply:
I did that hastily, resulting in a copying the previous elaboration. Now, I have corrected it. Thank you.